dashboard/admin.py

```diff
@@ -54,15 +66,18 @@ class CourseAdmin(admin.ModelAdmin):
     readonly_fields = ('term',)
 
     # Need this method to correctly display the line breaks
-    def _courseviewoption(self, obj):
+    @staticmethod
+    @admin_attr_decorator
+    def _courseviewoption(obj: Course) -> mark_safe:
         return mark_safe(linebreaksbr(obj.courseviewoption))
     _courseviewoption.short_description = "Course View Option(s)"
 
-    def course_link(self, obj):
+    @staticmethod
+    def course_link(obj: Course) -> format_html:
         return format_html('<a href="{}">Link</a>', obj.get_absolute_url())
 
     # When saving the course, update the id based on canvas id
-    def save_model(self, request, obj, form, change):
+    def save_model(self, request: HttpRequest, obj: Course, form: ModelForm, change: AlwaysChangedModelForm) -> super:
         obj.id = canvas_id_to_incremented_id(obj.canvas_id)
         return super(CourseAdmin, self).save_model(request, obj, form, change)
```

Comment:
This seems like it's returning `<class 'NoneType'>`, which makes sense. I don't think this method should have a return type; it's just saving the data.

Reply:
Yup, makes sense